### DISCUSSION TITLE
Migration from v5 to v6

### DISCUSSION THREAD

**Discussion by thryv-mallik** [webdriverio/codemod#79] [General]
Hi, I need to migrate from v5 to v6 and then to the latest. My project has three files: 1) wdio.shared.conf.js, 2) wdio.android.conf.js and 3) wdio.ios.conf.js .

The migration doc has this command: npx jscodeshift -t ./node_modules/@wdio/codemod/v6 ./wdio.conf.js. 
The above command contains **./wdio.conf.js**. As my project contains three files, do I need to use: wdio.shared.conf.js in the above command?

Kindly suggest.

Thanks,
Mallik

**Comment by christian-bromann** [comment]
> do I need to use: wdio.shared.conf.js in the above command?

Yes

**Reply by thryv-mallik** [reply]
Thank you for your reply. Getting the error below after giving this command:
Command: npx jscodeshift -t ./node_modules/@wdio/codemod/v6 ./wdio.shared.conf.js

ERROR Transform file /Users/ojas/Documents/node_modules/@wdio/codemod/v6 does not exist

**Reply by christian-bromann** [reply]
Can you check whether `/Users/ojas/Documents/node_modules/@wdio/codemod/v6` exists on your machine?

**Reply by thryv-mallik** [reply]
> Can you check whether `/Users/ojas/Documents/node_modules/@wdio/codemod/v6` exists on your machine?

Its not exist on my machine

**Reply by christian-bromann** [reply]
soo, did you install this codemod as explained [in the docs](https://github.com/webdriverio/codemod#install)?

**Reply by thryv-mallik** [reply]
Thank you, now able to see the codemod.
